### ISSUE TITLE
[Change Request]: Split A3A_fnc_fastTravelRadio

### ISSUE THREAD

**Issue by doommetal** [official-antistasi-community/A3-Antistasi#1792]
**Is your feature request related to a problem? Please describe.**
Needed for UI rework. Makes information about fast travel state able to be displayed as a tooltip on the fast travel button.

**Describe the solution you'd like**
Split the function into two parts:
1. A3A_fnc_canFastTravel: Checking if fast travel is available, should return a boolean and an array of reasons if false
2. A3A_fnc_fastTravel: Everything from beyond the availability check

**Describe alternatives you've considered**
We could leave the function as is though this means either no UI feedback on why you can't fast travel (leaving it with the hintboxes) or it requires duplicate code for the checks to be able to be displayed in the UI.

**Additional context**
![fasttravel](https://user-images.githubusercontent.com/4264673/111470999-b7619780-8728-11eb-8ab6-dbb42d77f832.png)

**Comment by doommetal** [comment]
tldr: Hold off on this one for now, will be done on UI rework merge.

I've looked a bit more into this one now and it's a bit of a mess:
* Group members in boats are supposed to block FT but you can still select a destination and wait through the travel countdown but nothing happens after that
* The above also applies for static weapons
* Some checks are done multiple times as they are needed both before and after the map is open (commander, high command mode etc.), preventing a simple split to canFT and FT.

I landed on integrating it more into the UI instead of just reusing the old FT, so it gets it's own map in the Y-menu itself, similar to the HC squad management and fire missions.
